frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -935,6 +935,35 @@ const cardList = handleActions(
   null,
 );
 
+function syncParametersAndEmbeddingParams(before, after) {
```

Comment:
Mind adding a couple unit tests for this? Add `export` to this line then in `frontend/test/metabase/dashboard/dashboard.unit.spec.js` add

```js
describe("syncParametersAndEmbeddingParams", () => {
  it("should remove `embedding_parameters` that are removed from `parameters`", () => {

  })
})
```

etc